quality_control_stock_oca/__manifest__.py

```diff
@@ -0,0 +1,24 @@
+# Copyright 2014 Serv. Tec. Avanzados - Pedro M. Baeza
+# Copyright 2014 Oihane Crucelaegui - AvanzOSC
+# Copyright 2018 Simone Rubino - Agile Business Group
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+{
+    "name": "Quality control - Stock (OCA)",
+    "version": "17.0.1.0.0",
+    "category": "Quality control",
+    "license": "AGPL-3",
+    "author": "OdooMRP team, AvanzOSC, Serv. Tecnol. Avanzados - Pedro M. Baeza, "
```

Comment:
One last thing. Please change this that they are old references:

```suggestion
    "author": "Tecnativa, AvanzOSC, "
```

Reply:
Done!